index.js

```diff
@@ -5,19 +5,26 @@ const eyeglass = require("eyeglass");
 
 const app = express()
 const port = 1337
+const debug = true
+const proxy = debug
```

Comment:
Would be nice if this wasn't on by default, otherwise how would we have found #111 -

Reply:
It is only enabled when debug is enabled and when the window.location is not hackmud.com.

If we want debug and proxy to be configurable, it probably makes sense to extract them to a config file before making them default false.

Reply:
There are a handful of options. Here are a few:
1. modifying package.json (and reading config out of there)
1. new .json file in the root
1. new config.js file that is referenced in the file via package require (allows for programmatic stuff to happen more than pure key-values)

I think we should probably do 2. for now.